# fix python version evaluation with more than 3 characters.
# orignal works with 3.8 but not with 3.10
# 
--- configure.org	2020-09-03 12:16:02.000000000 +0000
+++ configure	2022-03-27 16:13:00.017380651 +0000
@@ -19154,7 +19154,7 @@
 if ${am_cv_python_version+:} false; then :
   $as_echo_n "(cached) " >&6
 else
-  am_cv_python_version=`$PYTHON -c "import sys; sys.stdout.write(sys.version[:3])"`
+  am_cv_python_version=`$PYTHON -c "import sys; print(f'{sys.version_info.major}.{sys.version_info.minor}')"`
 fi
 { $as_echo "$as_me:${as_lineno-$LINENO}: result: $am_cv_python_version" >&5
 $as_echo "$am_cv_python_version" >&6; }
